docusaurus-plugin-moonwave/src/components/LuaType.js

```diff
@@ -181,18 +221,48 @@ function Token({ token, depth }) {
     case "union":
       return <Op>&nbsp;|&nbsp;</Op>
     case "luaType":
+      // Checks if the type is in the list of LuaClasses for the package
+      if (luaClassNames.includes(token.luaType.replaceAll("...", ""))) {
+        return (
+          <code className={styles.blue}>
+            <Link
+              style={{ textDecoration: "underline", color: "inherit" }}
+              to={`${baseUrl}api/${token.luaType.replaceAll("...", "")}`}
```

Comment:
I think it would be easier if we normalize this data structure. Right now we pass `robloxTypes` and `luaClassNames` separately, and they are different shapes.

We should just make one big map of Name->URL that we link to. This will simplify this logic here as well, because we don't have to have two branches. 

You would just build up this structure by adding to it for all Roblox classes, lua classes, and lua class types, in that order.

It should just be one docusaurus "data", call it `typeLinks` or something like that.